src/js/containers/covid19/Covid19Container.jsx

```diff
@@ -113,11 +114,16 @@ const Covid19Container = () => {
         const getAllSubmissionDates = async () => {
             allSubmissionDatesRequest.current = fetchAllSubmissionDates();
             try {
-                const { submissions } = await allSubmissionDatesRequest.current.promise.data;
-                dispatch(setLatestSubmissionDate(submissions.map((s) => moment(s)).sort((a, b) => a.value - b.value)[0].format('MMM DD[,] YYYY')));
+                const data = await allSubmissionDatesRequest.current.promise;
+                dispatch(setLatestSubmissionDate(data.data.available_periods
```

Comment:
This looks like pretty significant parsing from the api, can we put this into a helper so it can be tested?

Reply:
I tried but i keep getting this error, you seen this before?
<img width="812" alt="Screen Shot 2020-07-13 at 5 07 50 PM" src="https://user-images.githubusercontent.com/14301977/87353932-b6d68880-c52b-11ea-9c4d-4b9f0be708b9.png">

Reply:
@jonhill13 No let me look into it.

Reply:
This appears to fix it:
![image](https://user-images.githubusercontent.com/12897813/87446490-5ac14200-c5c7-11ea-99da-5792f563b12b.png)

[Pull from here](https://github.com/fedspendingtransparency/usaspending-website/tree/config/dev-5476/jest-config-changes) if you want or just add it yourself.